<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewLexDialog</class>
 <widget class="QDialog" name="NewLexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New lexical entry</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="explanation">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>10</y>
     <width>330</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Provide the information required to start a new entry.
Additional information can be added in the tab view.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>380</width>
     <height>85</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>85</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>85</height>
    </size>
   </property>
   <property name="title">
    <string>Metadata</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>26</y>
      <width>234</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speaker</string>
    </property>
   </widget>
   <widget class="QComboBox" name="speakerCode">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>46</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>26</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Researcher</string>
    </property>
   </widget>
   <widget class="QComboBox" name="researcherCode">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>46</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>77</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>77</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>380</width>
     <height>142</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>142</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>142</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="title">
    <string>Headword and definition</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>89</y>
      <width>34</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>34</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>34</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gloss</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="entryWord">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>29</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="gloss">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>60</height>
     </size>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>37</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Headword</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>306</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
